components/Text/NumberLabel.tsx

```diff
@@ -1,8 +1,8 @@
-import styled, { css } from 'styled-components';
+import styled from 'styled-components';
 
 type Contrast = 'strong' | 'mild';
 
-const sharedStyles = css<{ fontWeight?: string; contrast?: Contrast }>`
+export const NumberDiv = styled.div<{ fontWeight?: string; contrast?: Contrast }>`
```

Comment:
We should probably delete this and use our newer text components instead but maybe one for further refactor of text things.

Reply:
Yeah. We probably should. I'll make a note for it.